core/src/main/kotlin/org/jetbrains/research/testspark/core/generation/llm/LLMWithFeedbackCycle.kt

```diff
@@ -270,7 +270,9 @@ class LLMWithFeedbackCycle(
                 onWarningCallback?.invoke(WarningType.COMPILATION_ERROR_OCCURRED)
 
                 nextPromptMessage =
-                    "I cannot compile the tests that you provided. The error is:\n${testSuiteCompilationResult.second}\n Fix this issue in the provided tests.\nGenerate public classes and public methods. Response only a code with tests between ```, do not provide any other text."
+                    "I cannot compile the tests that you provided. The error is:\n${testSuiteCompilationResult.executionMessage}" +
+                            "\n Fix this issue in the provided tests.\nGenerate public classes and public methods. Response only a code with tests between ```," +
+                            " do not provide any other text."
```

Comment:
You can do:
````kotlin
nextPromptMessage = """
  I cannot compile the tests that you provided. The error is:
  ```
  ${testSuiteCompilationResult.executionMessage}
  ```

   Fix this issue in the provided tests.\nGenerate public classes and public methods. Response only a code with tests between ```, do not provide any other text.
""".trimIndent()
````